lib/rucio/core/replica.py

```diff
@@ -1753,20 +1753,25 @@ def get_sum_count_being_deleted(rse_id, session=None):
 
 
 @transactional_session
-def update_replicas_states(replicas, nowait=False, session=None):
+def update_replicas_states(replicas, nowait=False, add_tombstone=False, session=None):
```

Comment:
I would suggest to have`'tombstone=None` instead in the signature. If not set (==None), then it should be like the `add_tombstone=False` behaviour. If set, use this value for the tombstone instead ,e.g., `tombstone=datetime.utcnow() + timedelta(hours=2)`.

Reply:
Sounds reasonable. We can add this in a later patch.